typedoc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "inputFiles": ["src/*"],
+  "exclude": ["src/*/*/*", "src/globals.js", "src/registerServiceWorker.js"],
```

Comment:
What is `src/*/*/*`?

Can we include `globals.js`? My previous comment about removing `Globals` was the namespaces that it added to everything, not the globals file and its exports.

Reply:
I used `src/*/*/*` to exclude all files in second level subdirectories of src (e.g. components/Tutorial, components/__tests__).  Are they necessary for documentation?

Reply:
Let's exclude `tests`, but document the rest.